lib/audited.rb

```diff
@@ -13,12 +13,13 @@ class << self
     attr_writer :audit_class
```

Comment:
Perhaps you should replace `attr_writer` with a manual implementation to avoid the overhead in the getter:
```ruby
def audit_class=(cls)
  @audit_class = cls.is_a?(String) ? cls.constantize : cls
end
```

Reply:
I had it like this in my first draft but I think it would not help.
The constantize would be called directly on the setting (in the initializer).

I think it would be the same as writing: `Audited.audit_class = "CustomAudit".constantize`

Reply:
Oh, that's an interesting thing. Perhaps a comment in the `audit_class` why it's there (like "it has to be lazy") could help future readers.